src/lib/y2storage/proposal/autoinst_devices_creator.rb

```diff
@@ -62,11 +62,16 @@ module Proposal
     class AutoinstDevicesCreator
       include Yast::Logger
 
+      # @return [AutoinstIssues::List] List of found AutoYaST issues
+      attr_reader :issues_list
+
       # Constructor
       #
       # @param original_graph [Devicegraph] Devicegraph to be used as starting point
-      def initialize(original_graph)
+      # @param issues_list [AutoinstIssues::List] List of AutoYaST issues to register them
```

Comment:
documentation of parameter is not much clear. Maybe mention that it is write only parameter that is modified and issues found during creating device is added there.